packages/phone-number-privacy/combiner/src/config.ts

```diff
@@ -29,13 +29,13 @@ export interface OdisConfig {
   shouldFailOpen: boolean // TODO (https://github.com/celo-org/celo-monorepo/issues/9862) consider refactoring config, this isn't relevant to domains endpoints
   odisServices: {
     signers: string
-    timeoutMilliSeconds: number // XXX (soloseng): what is this timeout used for? could it also be used to replace FULL_NODE_TIMEOUT_IN_MS?
+    timeoutMilliSeconds: number
   }
   keys: {
     currentVersion: number
     versions: string // parse as KeyVersionInfo[]
   }
-  timeoutMs: number
+  fullNodeTimeoutMs: number
```

Comment:
you can get rid of this now in favor of where you've defined it in BlockchainConfig

Reply:
In the end I left it here instead of specifying in `blockchainConfig`, as we use the `OdisConfig.fullNodeTimeoutMs` when calling `authenticateUser()` in PNP*IO